src/epsilon/analyzer.d

```diff
@@ -918,15 +923,18 @@ void Analyse(Input input)
     if (ErrorCounter == 0)
     {
         EAG.History |= EAG.analysed;
-        info!"%s grammar is valid"(EAG.BaseName);
+        if (!reportOffset)
+            info!"%s grammar is valid"(EAG.BaseName);
     }
     else
     {
         EAG.History &= ~EAG.analysed;
-        if (NameNotified)
-            error!"errors in %s"(EAG.BaseName);
-        else
-            error!"errors";
+        if (!reportOffset) { // skip final message as language server intergration does not need them  
```

Comment:
I asked myself whether the log level for this message should have been "info".
If this would help with the language server, then we should change it.

Reply:
Yes, should be "info". It's just a summary at the end. I guess, it was added originally, to not oversee the errors in case a lot of warning are trailing. But for LSP it is a problem, because it sets an obvious marker at pos (1,1) that there are errors, what is obvious due to the other real markers.